sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/EventProcessor.cs

```diff
@@ -9,6 +9,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Azure.Core;
+using Azure.Messaging.EventHubs.Core;
```

Comment:
@kinelski:   See anything there that's going to cause a big problem with your incoming changes?    Any way that we can make the merge that we'll need easier, do you think?

Reply:
No, we're good. There aren't that many changes here.